client/iothub/model.go

```diff
@@ -88,6 +88,15 @@ const (
 	StatusDisabled Status = "disabled"
 )
 
+func NewStatusFromMenderStatus(status model.Status) Status {
```

Comment:
Nitpick: please consider differentiating Mender device status from Azure device status

Reply:
@MaciejTe feel free to address this in your PR